jest-setup.js

```diff
@@ -76,10 +85,29 @@ global.testRule = (rule, schema) => {
               if (testCase.column !== undefined) {
                 expect(_.get(warning, "column")).toBe(testCase.column)
               }
+
+              if (!schema.fix) return
+
+              if (!testCase.fixed) {
+                throw new Error("If using { fix: true } in test schema, all reject cases must have { fixed: .. }")
+              }
+
+              // Check the fix
+              return stylelint(Object.assign({ fix: true }, options)).then((output) => {
+                const fixedCode = getOutputCss(output)
+                expect(fixedCode).toBe(testCase.fixed)
+              })
             })
           })
         })
       })
     }
   })
 }
+
+function getOutputCss(output) {
+  const result = output.results[0]._postcssResult
+  return result.root.toString(result.opts.syntax)
+    // Less needs us to manually strip whitespace at the end of single-line comments ¯\_(ツ)_/¯
+    .replace(/(\n?\s*\/\/.*?)[ \t]*(\r?\n)/g, "$1$2")
```

Comment:
Let's narrow this function to Less syntax only. I. e. apply this transformation only if test case has `syntax: "less"`. This function could potentially mess with our tests results and give us false information.